Source/WebCore/Modules/webxr/WebXRSession.cpp

```diff
@@ -77,6 +77,13 @@ WebXRSession::WebXRSession(Document& document, WebXRSystem& system, XRSessionMod
     // https://immersive-web.github.io/webxr/#ref-for-dom-xrreferencespacetype-viewer%E2%91%A2
     // Every session MUST support viewer XRReferenceSpaces.
     device.initializeReferenceSpace(XRReferenceSpaceType::Viewer);
+
+    // https://www.w3.org/TR/webxr/#minimum-near-clip-plane
+    // Each XRSession has a minimum near clip plane and a maximum far clip plane, defined
+    // in meters. The values MUST be determined by the user agent and MUST be non-negative.
+    auto minimumNearClipPlaneFromPlatform = device.minimumNearClipPlane();
+    if (minimumNearClipPlaneFromPlatform >= 0)
+        m_minimumNearClipPlane = minimumNearClipPlaneFromPlatform;
```

Comment:
A bit late, but the preferred idiom in WebKit I think would be

m_minimumNearClipPlane = std::max(0, device.minimumNearClipPlane());